src/utils/csv.ts

```diff
@@ -0,0 +1,133 @@
+import { Aggregate, DatapointsGetAggregateDatapoint } from '@cognite/sdk';
+import moment from 'moment-timezone';
+import { getGranularityInSeconds } from './utils';
+
+export interface DatapointsToCSVProps {
+  data: DatapointsGetAggregateDatapoint[];
+  granularity: string;
+  aggregate: Aggregate;
+  delimiter: string;
+  format?: string;
+}
+
+export function datapointsToCSV({
+  data,
+  granularity,
+  aggregate = 'average',
+  delimiter = ',',
+  format,
+}: DatapointsToCSVProps): string {
+  const separator = delimiter === 'tab' ? '\t' : delimiter;
+  const ids = data.map(({ id, externalId }) => externalId || id.toString());
+  const rest = generateCSVStrings({
+    data,
+    granularity,
+    aggregate,
+    delimiter: separator,
+    format,
+  });
+
+  return [['timestamp', ...ids].join(separator), ...rest].join('\n');
+}
+
+export function downloadCSV(src: string, filename: string = 'noname.csv') {
```

Comment:
```suggestion
export function downloadCSV(src: string, filename: string = 'data.csv') {
```
to be consistent with what they have in opint

Reply:
done